wxCurrentPopupMenu is only defined if wxUSE_MENUS==1, so don't try to use it
if wxUSE_MENUS==0
---
 src/common/wincmn.cpp |    2 ++
 1 files changed, 2 insertions(+)

Index: b/src/common/wincmn.cpp
===================================================================
--- a/src/common/wincmn.cpp
+++ b/src/common/wincmn.cpp
@@ -317,10 +317,12 @@ wxWindowBase::~wxWindowBase()
     // we weren't a dialog class
     wxTopLevelWindows.DeleteObject((wxWindow*)this);
 
+#if wxUSE_MENUS
     // The associated popup menu can still be alive, disassociate from it in
     // this case
     if ( wxCurrentPopupMenu && wxCurrentPopupMenu->GetInvokingWindow() == this )
         wxCurrentPopupMenu->SetInvokingWindow(NULL);
+#endif // wxUSE_MENUS
 
     wxASSERT_MSG( GetChildren().GetCount() == 0, wxT("children not destroyed") );
 

 	  	 
